contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -66,27 +163,58 @@ NXmpes(NXobject):
         incident_polarization(NX_NUMBER):
           exists: recommended
           unit: NX_ANY
+        extent(NX_FLOAT):
+          exists: recommended
+        associated_source(NXsource):
+          doc: |
+            The source that emitted this beam.
+            Should be named with the same appendix, e.g.,
+            for `beam_probe` it should refer to `source_probe`.
+            Refers to the same concept as /NXentry/NXinstrument/source_TYPE
+            and may be linked.
       (NXelectronanalyser):
-        description:
-        energy_resolution(NX_FLOAT):
+        device_information(NXfabrication):
           exists: recommended
+          vendor:
+            exists: recommended
+          model:
+            exists: recommended
+          identifier:
+            exists: recommended
+        description:
+        work_function(NX_FLOAT):
           unit: NX_ENERGY
-          doc: |
-            Energy resolution of the analyser with the current setting. May be linked from a
-            NXcalibration.
-        fast_axes(NX_CHAR):
+          exists: recommended
+        energy_resolution(NXresolution):
+          exists: recommended
+          type:
+            exists: recommended
+          physical_quantity:
+            enumeration: [energy]
+          resolution(NX_FLOAT):
+        fast_axes:
           exists: recommended
         slow_axes:
           exists: recommended
+        transmission_function(NX_FLOAT):
```

Comment:
transmission_function is NXdata in NXelectronanalyzer

Reply:
Right, good catch. Changed to NXdata